InstallationGuide.md

```diff
@@ -34,6 +34,15 @@ The values are on the [Vault](https://vault.vault.demo.catena-x.net/).
 | dft.apiKey       | DFT_APIKEY                | url authentication code for edc asset payload | someCode      |
 | edc.enabled      | EDC_ENABLED               | enable / disable edc                          | true / false  |
 
+#### Keycloak variables:
+| Property name                     | Environment Variable Name | Description                  | Example Value |
+|-----------------------------------|---------------------------|------------------------------|---------------|
+| connector.discovery.token-url     | KEYCLOCK_HOSTNAME         | keyclock hostnam             | https://      |
+| connector.discovery.clientId      | KEYCLOCK_CLIENTID         | keyclock clientId            | X-Api_Key     |
+| connector.discovery.clientSecret  | KEYCLOCK_CLIENTSECRET     | keyclock clientse            | 123456        |
+| portal.backend.hostname           | PORTAL_HOSTNAME           | portal hostname              | https://      |
+
+
 The values are in the [Vault](https://vault.vault.demo.catena-x.net/).
```

Comment:
No one out there in the wild will have access to consortia internal vault instance. 

Please correct your `docker pull ...` command at the beginning of this documentation. The product lives in eclipse-tractusx (and the application image for a certain amount of time in the fork, hosted in the catenax-ng organization). And for what reason do you provide a Helm chart?!

Reply:
Suggested changes made. 
Removed the URL.